ansys_optical_automation/application/camera_nhtsa_test.py

```diff
@@ -0,0 +1,193 @@
+import os
+
+from ansys_optical_automation.speos_process.speos_sensors import Camera
+from ansys_optical_automation.speos_process.speos_simulations import Simulation
+
+
+def import_car(path, position_value):
+    param = GetActiveWindow().Groups[0]
+    param.SetDimensionValue(MM(position_value))
+    name = "Placement_position"
+    target = Selection.CreateByNames(name).Filter[ICoordinateSystem]()
+    target.SetActive()
+    DocumentInsert.Execute(path)
+
+
+def define_camera(
+    my_cam,
+    position,
+    focal_length=0.8,
+    imager_distance=10,
+    f_number=20,
+    distortion="OPTIS_Distortion_150deg.OPTDistortion",
+    transmittance="ANSYS_Transmittance.spectrum",
+    horz_pixel=1280,
+    vert_pixel=1280,
+    width=1.8,
+    height=1.8,
+    sensitivity=[
+        "ANSYS_Sensitivity_Red.spectrum",
+        "ANSYS_Sensitivity_Green.spectrum",
+        "ANSYS_Sensitivity_Blue.spectrum",
+    ],
+):
+    """
+    function to define the camera
+    Parameters
+    ----------
+    my_cam
+    position
+    focal_length
+    imager_distance
+    f_number
+    distortion
+    transmittance
+    horz_pixel
+    vert_pixel
+    width
+    height
+    sensitivity
+
+    Returns
+    -------
+
+
+    """
+    my_cam.set_position(
+        False,
+        False,
+        position.Items[0],
+        [
+            position.Items[0].GetDescendants[ICoordinateAxis]()[0],
+            position.Items[0].GetDescendants[ICoordinateAxis]()[1],
+        ],
+    )
+    my_cam.speos_object.FocalLength = focal_length
+    my_cam.speos_object.ImagerDistance = imager_distance
+    my_cam.speos_object.FNumber = f_number
+    my_cam.set_distortion(distortion)
+    my_cam.set_transmittance(transmittance)
+    my_cam.speos_object.HorzPixel = horz_pixel
+    my_cam.speos_object.VertPixel = vert_pixel
+    my_cam.speos_object.Width = width
+    my_cam.speos_object.Height = height
+    my_cam.set_sensitivity("red", sensitivity[0])
+    my_cam.set_sensitivity("green", sensitivity[1])
+    my_cam.set_sensitivity("blue", sensitivity[2])
```

Comment:
@BenoitHeraud I believe should methods can be better organized under Camera class